src/Catty/DataModel/VariablesContainer/UserDataContainer.m

```diff
@@ -218,150 +158,73 @@ - (void)removeProjectUserVariableNamed:(NSString*)name
 - (void)removeProjectUserListNamed:(NSString*)name
```

Comment:
Please remove this deprecated method and remove the variable directly at `removeUserListWithName`.